src/libtexture/imagecache_pvt.h

```diff
@@ -263,7 +263,13 @@ class OIIO_API ImageCacheFile : public RefCnt {
         bool is_constant_image;         ///< Is the image a constant color?
         bool has_average_color;         ///< We have an average color
         std::vector<float> average_color; ///< Average color
-        spin_mutex average_color_mutex; ///< protect average_color
+        spin_mutex mutex;               ///< protect average_color and Mlocal
+        std::unique_ptr<Imath::M44f> Mlocal; ///< shadows/volumes: world-to-local
+#if USE_SHADOW_MATRICES
```

Comment:
Not really related to this review, but do you think we'll ever implement shadow map lookups?

Reply:
Probably not? In the ancient past, I thought we would so I made room, but then never implemented the calls. Then I #ifdef'ed out the storage since we weren't using it.

I guess I should probably just remove it now. It's not like there's a lot to re-add if we change our minds later.